### ISSUE TITLE
Negative Progress

### ISSUE THREAD

**Issue by creat89** [crossminer/scava#279]
Hello,

For unknown reasons the progress bar showed me a negative progress:
![image](https://user-images.githubusercontent.com/17121539/59099426-f9866400-891a-11e9-9928-14d5c8129c97.png)
There were some issues during the running, but, still I wonder why the progress was negative.

**Comment by blueoly** [comment]
I have noticed something similar happening but only in the first moments of the analysis. Initially, the value of the progress bar is -25% and after 1-2 seconds it starts acting normally.

**Comment by valeriocos** [comment]
Hi @blueoly @creat89 this issue is probably related to https://github.com/crossminer/scava/issues/276

**Comment by creat89** [comment]
@valeriocos no. I don't think it is related as at the end the task finished completely without any other issue. Also, I'm not analyzing a project related to vcs but communication channels.
